cmd/lncli/profile.go

```diff
@@ -36,6 +36,10 @@ type profileEntry struct {
 
 // cert returns the profile's TLS certificate as a x509 certificate pool.
 func (e *profileEntry) cert() (*x509.CertPool, error) {
+	if e.TLSCert == "" {
+		return nil, nil
```

Comment:
Define some error `ErrTLSNotSet` instead?

Reply:
I think the `nil` is fine too. There is no pool, so we return `nil`.